GeneralsMD/Code/Main/WinMain.cpp

```diff
@@ -436,7 +432,7 @@ LRESULT CALLBACK WndProc( HWND hWnd, UINT message,
 			case WM_KILLFOCUS:
 			{
 				if (TheKeyboard )
```

Comment:
nit: While here, could we go with spaces or no spaces for (TheKeyboard) and (TheMouse), this one in particular only has the space on one side